src/foam/box/RPCErrorMessage.js

```diff
@@ -23,6 +23,13 @@ foam.CLASS({
     {
       class: 'Object',
       name: 'data'
+    },
+    {
+      class: 'String',
+      name: 'message',
+      expression: function(data) {
```

Comment:
More efficient in this case to just implement getter: since nobody would ever want to set this directly. Also, make transient since we don't need to send across the network.